tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -911,12 +912,15 @@ <h3>There’s no dashboard by the name of “<tt>[[_selectedDashboard]]</tt>.”
         this.$.settings.open();
         this.$.paginationLimitInput.value = tf_paginated_view.getLimit();
       },
+      _paginationLimitValidate(event) {
+        event.target.validate();
+      },
       _paginationLimitChanged(e) {
-        const value = e.target.valueAsNumber;
+        const value = Number.parseInt(e.target.value, 10);
         // We set type="number" and min="1" on the input, but Polymer
         // doesn't actually enforce those, so we have to check manually.
         if (value === +value && value > 0) {
```

Comment:
Is this `value === +value` just `!isNaN(value)` now?

Reply:
Probably can remove it now but I honestly don't understand this type coercion and decided to keep it as is.